pkg/prompt/prompt.go

```diff
@@ -94,8 +94,8 @@ type Pipe struct {
 
 // Confirm asks for yes or no input
 func (u Pipe) Confirm(message string, defaultValue bool) (bool, error) {
-	fmt.Fprint(u.stdout, message)
+	_, _ = fmt.Fprint(u.stdout, message)
 	var answer string
-	fmt.Scanln(&answer)
+	_, _ = fmt.Scanln(&answer)
```

Comment:
This I wasn't sure about; perhaps could also require the error to be returned (but wasn't sure what kind of errors it returned)